### PR TITLE
DOCS-2350 updating upgrade doc.

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: anypoint-private-cloud/v/2.0/upgrade.adoc [hunk 1]
```diff
@@ -70,12 +70,12 @@ This command updates the platform to the current version and restarts each pod i
...
-synchronizing anypoint:1.7.1 with Docker registry at 172.31.0.45:5000
-synchronizing anypoint:1.7.1 with Docker registry at 172.31.0.5:5000
-anypoint:1.7.0 has been uploaded
-updating anypoint from 1.6.1 to 1.7.1
+importing anypoint:2.0
+synchronizing anypoint:2.0 with Docker registry at 172.31.0.26:5000
+synchronizing anypoint:2.0 with Docker registry at 172.31.0.45:5000
+synchronizing anypoint:2.0o with Docker registry at 172.31.0.5:5000
+anypoint:2.0 has been uploaded
+updating anypoint from 1.7.1 to 2.0
```
Comment (pablodcar) [inline anypoint-private-cloud/v/2.0/upgrade.adoc]: Now the output is different, this is an example

```Thu Aug 23 09:32:03 UTC	Importing application anypoint v2.0.0-alpha-266-g1133b2e
Thu Aug 23 09:36:30 UTC	Synchronizing application with Docker registry 10.1.3.213:5000
Thu Aug 23 09:44:27 UTC	Synchronizing application with Docker registry 10.1.5.247:5000
Thu Aug 23 09:51:29 UTC	Synchronizing application with Docker registry 10.1.1.81:5000
Thu Aug 23 09:58:23 UTC	Application has been uploaded
updating anypoint from 1.7.1 to 2.0.0-alpha-266-g1133b2e
update operation (d077277f-3c69-4336-82bf-bd8b428ae202) has been started
the cluster is updating in background
```

File: anypoint-private-cloud/v/2.0/upgrade.adoc [hunk 2]
```diff
@@ -97,3 +97,16 @@ $ kubectl get pods
...
 
+. Configure Anypoint Private Cloud to use your NFS server.
++
+After upgrading Anypoing Private Cloud, you configure it to use the required NFS server.
++
+.. From Anypoint Platform, select *Access Management*.
+.. Click *NFS Settings*.
+.. Enter the domain name of your NFS server.
+.. Enter the path to your NFS server.
+.. Click *Test NFS Server*.
```
Comment (pablodcar) [inline anypoint-private-cloud/v/2.0/upgrade.adoc]: First Save, once the save is done, test.

File: anypoint-private-cloud/v/2.0/upgrade.adoc [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
-| Base Version | Intermediate Version | Target Version | Comments
-| 1.7.0 | N/A | 1.7.1 | Direct migration is supported
-| 1.6.1 | N/A | 1.7.1 | Direct migration is supported
-| 1.6.0 | N/A | 1.7.1 | Direct migration is supported
-| 1.5.2 | 1.6.1 | 1.7.1 | Must upgrade to 1.6.1 first
-| 1.5.1 | 1.6.1 | 1.7.1 | Must upgrade to 1.6.1 first
+| Base Version | Intermediate Versions | Target Version | Comments
+| 1.7.x | N/A | 2.0 | Direct upgrade is supported
+| 1.6.x | 1.7.1 | 2.0 | Must upgrade to version 1.7.1
+| 1.5.x | 1.6.1, 1.7.1 | 2.0 | Must upgrade to 1.6.1, then 1.7.1 before upgrading to 2.0.
```
Comment (prafulrana) [inline anypoint-private-cloud/v/2.0/upgrade.adoc]: can update to 1.7.0 too.

File: anypoint-private-cloud/v/2.0/upgrade.adoc [hunk 4]
```diff
@@ -96,4 +101,20 @@ $ kubectl get pods
 ----
 https://{platform-host}:9500
 ----
++
+[WARNING]
+Before accessing Anypoint Platform, ensure that you configure your NSF server s described in the following step. Failure to configure NFS before accessing the platform may result in data loss.
```
Comment (thejsj) [inline anypoint-private-cloud/v/2.0/upgrade.adoc]: `NSF` -> `NFS`

Comment (thejsj) [reply anypoint-private-cloud/v/2.0/upgrade.adoc]: `s` -> `as`

File: anypoint-private-cloud/v/2.0/upgrade.adoc [hunk 5]
```diff
@@ -96,4 +101,20 @@ $ kubectl get pods
...
+. Configure Anypoint Private Cloud to use your NFS server.
++
+After upgrading Anypoing Private Cloud, you configure it to use the required NFS server.
++
+.. From Anypoint Platform, select *Access Management*.
+.. Click *NFS Settings*.
+.. Enter the domain name of your NFS server.
+.. Enter the path to your NFS server.
+.. Click *Save*.
+.. Click *Test NFS Server* to test that your NFS server is configured correctly.
```
Comment (thejsj) [inline anypoint-private-cloud/v/2.0/upgrade.adoc]: @sebasampaoli actually changed this last minute. Now they should test the NFS server _before_ they save the credentials. @sebasampaoli, can you confirm?

File: anypoint-private-cloud/v/2.0/upgrade.adoc (hunk 3 above)
Comment (sleslie23) [reply anypoint-private-cloud/v/2.0/upgrade.adoc]: @prafulrana Maybe this isn't clear, but the idea of this table is that the version you are migrating from is the "Base Version." 1.7.1 is the version you have to migrate TO to get from 1.6.x to 2.0.

File: anypoint-private-cloud/v/2.0/upgrade.adoc (hunk 5 above)
Comment (sebasampaoli) [reply anypoint-private-cloud/v/2.0/upgrade.adoc]: Yes, thanks @thejsj . @sleslie23 I'll send you the new screenshoot soon.

File: anypoint-private-cloud/v/2.0/upgrade.adoc (hunk 4 above)
Comment (sleslie23) [reply anypoint-private-cloud/v/2.0/upgrade.adoc]: Fixed!

File: anypoint-private-cloud/v/2.0/upgrade.adoc (hunk 5 above)
Comment (sleslie23) [reply anypoint-private-cloud/v/2.0/upgrade.adoc]: According to @pablodcar this is the correct order.

Comment (sebasampaoli) [reply anypoint-private-cloud/v/2.0/upgrade.adoc]: Yes, I think you can keep that order. I sent you an image with the new NFS settings UI.

**Pull request merged**